test/server/index.js

```diff
@@ -101,6 +102,56 @@ SuperTokens.init({
+                        get: (redirectURI, authCodeFromRequest) => {
+                            return {
+                                accessTokenAPI: {
+                                    // this contains info about the token endpoint which exchanges the auth code with the access token and profile info.
+                                    url: `${process.env.AUTH0_DOMAIN}/oauth/token`,
+                                    params: {
+                                        // example post params
+                                        client_id: process.env.AUTH0_CLIENT_ID,
+                                        client_secret: process.env.AUTH0_CLIENT_SECRET,
+                                        grant_type: "authorization_code",
+                                        redirect_uri: redirectURI,
+                                        code: authCodeFromRequest,
+                                    },
+                                },
+                                authorisationRedirect: {
+                                    // this contains info about forming the authorisation redirect URL without the state params and without the redirect_uri param
+                                    url: `${process.env.AUTH0_DOMAIN}/authorize`,
+                                    params: {
+                                        client_id: process.env.AUTH0_CLIENT_ID,
+                                        scope: "openid profile",
+                                        response_type: "code",
+                                    },
+                                },
+                                getClientId: () => {
+                                    return process.env.AUTH0_CLIENT_ID;
+                                },
+                                getProfileInfo: async (accessTokenAPIResponse) => {
+                                    let accessToken = accessTokenAPIResponse.access_token;
+                                    let authHeader = `Bearer ${accessToken}`;
+                                    let response = await axios({
+                                        method: "get",
+                                        url: `${process.env.AUTH0_DOMAIN}/userinfo`,
+                                        headers: {
+                                            Authorization: authHeader,
+                                        },
+                                    });
+                                    let userInfo = response.data;
+                                    return {
+                                        id: userInfo.sub,
+                                        email: {
+                                            id: userInfo.name,
+                                            isVerified: true,
+                                        },
+                                    };
+                                },
+                            };
+                        },
+                    },
```

Comment:
Extract this into its own function and reuse it in third-party and thirdpartyemailpassword config..